web/source/osk/browser/keytip.ts

```diff
@@ -78,7 +78,7 @@ namespace com.keyman.osk.browser {
         let canvasWidth = xWidth + Math.ceil(xWidth * 0.3) * 2;
         let canvasHeight = Math.ceil(2.3 * xHeight) + (ySubPixelPadding); //
 
-        kts.top = Math.floor(y - canvasHeight) + 'px';
+        kts.bottom = Math.floor(keyman.osk.computedHeight - y) + 'px';
```

Comment:
Why does this fix the issue?

https://github.com/keymanapp/keyman/blob/14b14857728d55948fa4b24b3ab67ac8438687f9/web/source/osk/browser/keytip.ts#L116-L119

Line 118 was broken - as it turns out, `cs.bottom == 'auto'` in our currently-released version.  That doesn't convert nicely to a number... causing `bottomY == NaN`.  That introduces a _lovely_ set of issues, as you can imagine.

https://github.com/keymanapp/keyman/blob/14b14857728d55948fa4b24b3ab67ac8438687f9/web/source/osk/browser/keytip.ts#L128-L129

The problem is, we _need_ `bottomY` for our calculations that constrain the keytip within bounds.  So, the best answer:  flip from a "top"-based approach to a "bottom"-based one.

-----

After a bit of analysis, I determined `y` to be "the distance of the key tip's bottom from the _top_ of its parent element."  Thing is, for CSS `.bottom` use, we need that to be "... from the _bottom_ of its parent element."  Fortunately, a simple calc.
-  Before, for use of `.top`, it needed "the distance of the key tip's top...".